packages/graphql-client/src/utils/TestUtils.ts

```diff
@@ -0,0 +1,5 @@
+export class TestUtils {
```

Comment:
I think a class is not necessary here, you can export mockResponse directly. Also, files should be named using camelCase.

Reply:
Done and done!